crates/driver/src/commit_reveal.rs

```diff
@@ -7,8 +9,17 @@ use solver::{
 
 /// A `SolutionSummary` holds all information solvers are willing to disclose during settlement
 /// competition. It does **not** have to include the call data, yet.
-#[derive(Debug, Clone, Deserialize, Serialize)]
-pub struct SettlementSummary {}
+#[derive(Debug, Default, Clone, Deserialize, Serialize)]
+pub struct SettlementSummary {
+    /// Surplus is denominated in the chain's native token and based off of the auction's external
+    /// prices.
+    pub surplus: f64,
+    /// This is how much gas the solver would like to get reimbursed for executing this solution.
+    pub gas_reimbursement: U256,
```

Comment:
Does this mean that if solvers decide to not include the call data, they will get reimbursed only the gas they are quoting? And this amount will also be used when ranking the solutions?

Reply:
Yes, exactly.